README.md

```diff
@@ -112,3 +112,17 @@ $ gulp watch
 [16:50:20] Finished 'scripts:app' after 454 ms
 
 ```
+
+## Deploying to Cloud Foundry
```

Comment:
I'd change the paragraph title and description below to mention that it's an instruction how to push console to offline environment. I guess in most of the development cases these steps are not needed to be done before each push.